test-suite/src/like_ilike.rs

```diff
@@ -124,4 +131,70 @@ test_case!(like_ilike, async move {
     for (sql, error) in error_sqls {
         test!(sql, Err(error));
     }
+
+    test! {
+        name: "`_` should match any single character",
+        sql: r#"SELECT 'GLUE+SQL' LIKE 'GLUE_SQL' AS column1"#,
+        expected: Ok(select!(column1 Bool; true))
+    };
+
+    test! {
+        name: r#"`\` should escape `_` to normal character - false"#,
+        sql: r#"SELECT 'GLUE+SQL' LIKE 'GLUE\_SQL'"#,
+        expected: Ok(select!(column1 Bool; false))
+    };
+
+    test! {
+        name: r#"`\` should escape `_` to normal character - true"#,
+        sql: r#"SELECT 'GLUE_SQL' LIKE 'GLUE\_SQL'"#,
+        expected: Ok(select!(column1 Bool; true))
+    };
+
+    test! {
+        name: "`%` should match any sequence of zero or more characters",
+        sql: r#"SELECT 'GLUE+-SQL' LIKE 'GLUE%SQL'"#,
+        expected: Ok(select!(column1 Bool; true))
+    };
+
+    test! {
+        name: r#"`\` should escape `%` to normal character - false"#,
+        sql: r#"SELECT 'GLUE+-SQL' LIKE 'GLUE\%SQL'"#,
+        expected: Ok(select!(column1 Bool; false))
+    };
+
+    test! {
+        name: r#"`\` should escape `%` to normal character - true"#,
+        sql: r#"SELECT 'GLUE%SQL' LIKE 'GLUE\%SQL'"#,
+        expected: Ok(select!(column1 Bool; true))
+    };
+
+    test! {
+        name: r#"`\` should escape `\` to normal character"#,
+        sql: r#"SELECT 'GLUE\SQL' LIKE 'GLUE\\SQL'"#, ////
+        expected: Ok(select!(column1 Bool; true))
+    };
+
+    test! {
+        name: r#"`\` should escape `'` to normal character"#,
+        sql: r#"SELECT 'GLUE''SQL' LIKE 'GLUE\'SQL'"#,
+        expected: Ok(select!(column1 Bool; true))
+    };
```

Comment:
Currently, I realized that this SQL is not supported by sqlparser so that let's remove it for now.